app/views/coronavirus_local_restrictions/_tier_one_restrictions.html.erb

```diff
@@ -1,19 +1,23 @@
 <%
+  tier_heading_pretext = t("coronavirus_local_restrictions.results.level_one.heading_pretext")
+  tier_heading_label = t("coronavirus_local_restrictions.results.level_one.heading_tier_label")
+  tier_heading = "#{tier_heading_pretext} <span class=\"coronavirus-local-restriction__heading-tier-label\">#{tier_heading_label}</span>"
+
```

Comment:
This feels like a lot of logic. As `tier_heading_pretext` and `tier_heading_label` are not used anywhere else in the view would it be easier to just have one assignment? 

Also, rather interpolating CSS in a string, which feels weird, could the `tier_heading` be assigned in a `capture` block like:

```
<% tier_heading = capture do %>
  <%= t("coronavirus_local_restrictions.results.level_one.heading_pretext") %>
  <span class="coronavirus-local-restriction__heading-tier-label">
    <%= t("coronavirus_local_restrictions.results.level_one.heading_tier_label") %>
  </span>
<% end %>
```

Reply:
Didn't know this - thanks Leena!